tests/test_pyseq.py

```diff
@@ -487,6 +489,40 @@ def test_get_sequences_is_working_properly_3(self):
                 seq.format('%h%p%t %r')
             )
 
+    def test_walk_succeed(self):
```

Comment:
Totally agree with that.  I was just following the current setup for testing files.